pkg/policy/selectorcache.go

```diff
@@ -106,11 +107,25 @@ type SelectorCache struct {
 	userMutex lock.Mutex
 	// userNotes holds a FIFO list of user notifications to be made
 	userNotes []userNotification
+	// notifiedUsers is a set of all notified userNotes
```

Comment:
it's the set of all notified users, right?

Reply:
correct! fixed